addons/html_editor/static/src/fields/html_field.js

```diff
@@ -32,14 +46,66 @@ export class HtmlField extends Component {
                 this.lastValue !== record.data[this.props.name].toString()
             ) {
                 this.state.key++;
+                this.state.containsComplexHTML = this.computeContainsComplexHTML();
             }
         });
+
+        useEffect(
+            () => {
+                if (this.iframeRef.el) {
+                    const onLoadIframe = () => this.onLoadIframe();
```

Comment:
maybe you can just use the onLoad from the wysiwyg component and not have to do all this manually

Reply:
It s not the wysiwyg use case. It s for the readonly mode